lib/webmachine/media_type.rb

```diff
@@ -112,11 +113,7 @@ def minor
     #   ignoring params and taking into account wildcards
     def type_matches?(other)
       other = self.class.parse(other)
-      if [Dispatcher::Route::MATCH_ALL_STR, MATCHES_ALL, type].include?(other.type)
-        true
-      else
-        other.major == major && other.minor == Dispatcher::Route::MATCH_ALL_STR
-      end
+      [Dispatcher::Route::MATCH_ALL_STR, MATCHES_ALL, type].include?(other.type) || other.major == major && other.minor == Dispatcher::Route::MATCH_ALL_STR
```

Comment:
This line is a bit long, can you please add a new line after the `||`?

Reply:
Updated. Thanks for your suggest